pkg/controller/daemon/daemon_controller.go

```diff
@@ -368,8 +368,8 @@ func (dsc *DaemonSetsController) getDaemonSetsForHistory(history *apps.Controlle
 	if len(daemonSets) > 1 {
 		// ControllerRef will ensure we don't do anything crazy, but more than one
 		// item in this list nevertheless constitutes user error.
-		klog.V(4).Infof("User error! more than one DaemonSets is selecting ControllerRevision %s/%s with labels: %#v",
-			history.Namespace, history.Name, history.Labels)
+		klog.V(4).InfoS("User error! More than one DaemonSet is selecting ControllerRevision",
+			"controllerRevision", klog.KObj(history))
```

Comment:
this loses the labels info that was previously logged... that seems like it would be important for tracking down the conflict

Reply:
@serathius what would be the correct way to handle this?